### DISCUSSION TITLE
Can add Chinese?

### DISCUSSION THREAD

**Discussion by fangbian233** [YimMenu/YimMenu#306] [Ideas]
Can Chinese be supported?pls

**Comment by Yimura** [comment]
There's no translation built into YimMenu so no.

**Reply by fangbian233** [reply]
I can help you translate Chinese

**Reply by fangbian233** [reply]
But I don't know how to change it

**Reply by Yimura** [reply]
Sure, I'll look into porting my translation system to YimMenu.

**Reply by fangbian233** [reply]
thanks
